react/ProductDetails.js

```diff
@@ -1,9 +1,9 @@
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
+import React, { useCallback } from 'react'
 import { mapObjIndexed, mergeDeepRight, path } from 'ramda'
 import { injectIntl, intlShape, FormattedMessage } from 'react-intl'
 
-import { ExtensionPoint, withRuntimeContext } from 'vtex.render-runtime'
+import { ExtensionPoint, useRuntime } from 'vtex.render-runtime'
```

Comment:
You need to create the mock of `useRuntime`. The tests are failing because of that.